Kernel/Config/Files/XML/Daemon.xml

```diff
@@ -738,7 +738,7 @@
             </Hash>
         </Value>
     </Setting>
-    <Setting Name="Daemon::SchedulerCronTaskManager::Task###SupportDataCollectAsynchronous" Required="1" Valid="1" ReadOnly="1" ConfigLevel="100">
+    <Setting Name="Daemon::SchedulerCronTaskManager::Task###SupportDataCollectAsynchronous" Required="0" Valid="0" ConfigLevel="100">
```

Comment:
```suggestion
    <Setting Name="Daemon::SchedulerCronTaskManager::Task###SupportDataCollectAsynchronous" Required="1" Valid="1" ConfigLevel="100">
```

We will not disable this SysConfig by default.
However, we will remove `ReadOnly`.

Reply:
If 'ReadOnly' is removed you can adapt to your needs.
If you have no more reason - We will not disable this SysConfig by default.